blockchain/v2/io.go

```diff
@@ -132,6 +133,24 @@ func (sio *switchIO) trySwitchToConsensus(state state.State, skipWAL bool) bool
 	return ok
 }
 
+func (sio *switchIO) sendStatusRequest(peerID p2p.ID) error {
```

Comment:
Maybe just pass the peer directly, so we don't have to look it up in the switch.